backend/src/main/java/de/neuefische/booklibrary/backend/Book.java

```diff
@@ -1,4 +1,7 @@
 package de.neuefische.booklibrary.backend;
 
+import lombok.With;
+
+@With
 public record Book(String id, String title, String author, String isbn, BookState bookState) {
```

Comment:
Using enums is nice, but if there are only 2 options (available and not), maybe just use a boolean "isAvailable"?